vhdbuilder/packer/vhd-image-builder-cvm.json

```diff
@@ -0,0 +1,691 @@
+{
+  "variables": {
+    "subscription_id": "{{env `AZURE_SUBSCRIPTION_ID`}}",
+    "cvm_packer_build_location": "{{env `CVM_PACKER_BUILD_LOCATION`}}",
```

Comment:
nit: this can just be `packer_build_location` and come from `PACKER_BUILD_LOCATION` like the other templates